Formula/libjwt.rb

```diff
@@ -0,0 +1,44 @@
+class Libjwt < Formula
+  desc "JWT C Library"
+  homepage "https://benmcollins.github.io/libjwt/"
+  url "https://github.com/benmcollins/libjwt/archive/v1.8.0.tar.gz"
+  sha256 "e97c15c14dbd49d5ba435cb26e06e6cdd6e32cf9f0be6b6e9a2d2d330923ea17"
+
+  depends_on "autoconf" => :build
+  depends_on "pkg-config" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "openssl"
+  depends_on "jansson"
+  depends_on "check" => :build
+
+  def install
+    system "autoreconf", "-v", "-i"
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "all"
+    system "make", "check"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <stdio.h>
+      #include <stdlib.h>
+      #include <errno.h>
+
+      #include <jwt.h>
+
+      int main() {
+        jwt_t *jwt = NULL;
+        int token = 0;
+
+        token = jwt_new(&jwt);
```

Comment:
`if (jwt_new(&jwt)) abort();` so that we shorten the test program :)

Reply:
Neat. Thanks.